lib/bot.rb

```diff
@@ -78,11 +78,13 @@ def pull_request_comment_content
   end
 
   def extract_qa_comment
-    search_pattern = /#{@magic_qa_keyword}.*\w+/i
+    search_pattern = /(.*)(#{@magic_qa_keyword})(.*\w+.*)/im
+
     return unless @comment.match?(search_pattern)
 
-    parts = @comment.partition search_pattern
-    parts[1] + parts[2]
+    _ignored_prefix, qa_keyword, qa_comment = @comment.match(search_pattern).captures
+
+    qa_keyword + qa_comment
```

Comment:
How about?
```ruby
def extract_qa_comment
  qa_comment = @comment[/#{@magic_qa_keyword}\s*(.*)/im, 1]
  return unless qa_comment

  "#{@magic_qa_keyword} qa_comment"
end
```

It does the match only once and makes the "QA:" prefix in Jira consistent, independent from what user provided(e.g. "Qa", "qa").